src/components/elements/accordion.tsx

```diff
@@ -93,15 +118,35 @@ const ItemExpanded = ({ is_default_open, child, child_idx, nodes, id }) => {
         else setHeight(0)
     })
 
-    React.useEffect(() => {
+    useEffect(() => {
         if (is_default_open) setExpanded(true)
     }, [])
 
-    React.useEffect(() => child && setHeight(getHeight(child_idx)), [is_expanded])
+    useEffect(() => child && setHeight(getHeight(child_idx)), [is_expanded])
+
+    const deployer = is_expanded ? (
+        <img src={Minus} alt="Minus" height="16" width="16" />
+    ) : (
+        <img src={Plus} alt="Plus" height="16" width="16" />
+    )
+
+    const expanded_state = is_expanded ? true : false
+
+    const current_arrow = child.props.arrow_thin ? (
+        <Arrow src={Chevron} alt="Chevron" width="32" height="32" expanded={expanded_state} />
+    ) : (
+        <ThickArrow
+            src={ChevronThick}
+            alt="Chevron thick"
+            width="32"
+            height="32"
+            expanded={expanded_state}
+        />
+    )
 
     return (
         <>
-            {child.props.is_showed != false && (
+            {child.props.is_showed !== false && (
```

Comment:
A suggestion for an improvement 🙂:
```suggestion
            {child.props.is_showed && (
```
*also, it would be nice to change is_showed to a grammatically correct is_shown if possible)